tests/PhoneValidatorTest.php

```diff
@@ -11,14 +11,14 @@ class PhoneValidatorTest extends TestCase
 
     protected $validator;
 
-    protected function getPackageProviders()
+    protected function getPackageProviders($app)
```

Comment:
Holding off on this. Omitting the `$app` parameter will generate a non-blocking warning for Laravel 5 builds, but allows to use the same tests file for Laravel 4.